index.js

```diff
@@ -17,8 +17,14 @@ function LessCompiler (sourceTrees, inputFile, outputFile, options) {
   if (!(this instanceof LessCompiler)) {
     return new LessCompiler(sourceTrees, inputFile, outputFile, options)
   }
+  
+  var cacheOptions = merge({
+    filterFromCache: {
+      include: [/.*\.less$/]
```

Comment:
actually this may screw up alternative files that do not end in less that must be considered for less compilation.  Does less do any image/font base64 inlining (or similar)